integration-tests/features/drift.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Drift Detection
+  Scenario: Detects no drift on a stack that with no drift
+    Given stack "14/A" exists using "loggroup14.yaml"
```

Comment:
Honestly, these numbered stack group names like "14/A" for integration tests are simply hard to read and understand what they're for and I think they ultimately lead to maintainability problems. If someone comes to the templates in group 14 a year from now, they'll have no idea what those templates are supposed to be testing. I know you care a lot about convention, but if it's any consolation, @zaro0508 has recently approved some PRs of mine for my project dependency work with explicitly named stack groups that make them a lot easier to read and understand:

* https://github.com/Sceptre/sceptre/pull/1153/files#diff-491c6350fe3e98371009887e70ca303f80d0470839d6e93d8606079965019fd1
* https://github.com/Sceptre/sceptre/pull/1154/files#diff-335da79f520e01230446d2c35297bc7ab5b806699ca6d2c3bc0a3b7b4dad6e7f
* https://github.com/Sceptre/sceptre/pull/1185/files#diff-491c6350fe3e98371009887e70ca303f80d0470839d6e93d8606079965019fd1

All that to say, there's a _better_ convention I think we can follow that is easier to understand and to work with.

Reply:
@jfalkenstein  What I think is good about conventions is they are there to stop individual team members from rewriting all the code according to their own personal preferences. After all, a convention is really just a preference. We do 4-spaces indenting in Python because that is a shared preference of the Python community that is not shared by the Ruby community that prefers 2-space indenting! With that said, I think you can argue objectively rather subjectively that `14/A` is a bad name. I think fixing that ought to go in a separate change that renames of these stacks using better names—that's how I would manage this. But considering you already have a PR open that I did not know about that changes it, I will refactor here to align to your new names!